tests/client/test_search_parameters.py

```diff
@@ -85,3 +87,14 @@ def test_courts(self, test_input, expected):
         """
         search_parameters = SearchParameters(court=test_input)
         assert set(search_parameters.as_marklogic_payload()["court"]) == set(expected)
+
+    def test_quoted_phrases(self):
```

Comment:
Can we test multiple quotes, like `"a b" c "d e"` as well please? Want to be sure `" c "` doesn't pop up

Reply:
aha good plan - I did a bunch of those tests in ipython and really should have put them in here :-)